suite-native/module-home/src/components/PortfolioContent.tsx

```diff
@@ -14,10 +15,15 @@ import {
 import { selectIsSelectedDeviceImported } from '@suite-common/wallet-core';
 import { useTranslate } from '@suite-native/intl';
 
-import { PortfolioGraph } from './PortfolioGraph';
+import { PortfolioGraph, PortfolioGraphRef } from './PortfolioGraph';
 
-export const PortfolioContent = () => {
+export type PortfolioContentRef = {
+    refetchGraph?: () => Promise<void>;
+};
+
+export const PortfolioContent = forwardRef<PortfolioContentRef>((_props, ref) => {
     const { translate } = useTranslate();
+    const graphRef = useRef<PortfolioGraphRef>(null);
```

Comment:
Shouldn't the ref type take null into account?

Reply:
It's handled internally because ref always can be null/undefined